Fabric.Authorization.Domain/Services/UserService.cs

```diff
@@ -8,10 +11,17 @@ namespace Fabric.Authorization.Domain.Services
     public class UserService
     {
         private readonly IUserStore _userStore;
+        private readonly IRoleStore _roleStore;
 
-        public UserService(IUserStore userStore)
+        public UserService(IUserStore userStore, IRoleStore roleStore)
```

Comment:
Do we need not null validations on both constructor parameters?

Reply:
I'll add those in.